test/integration/testQueryGraphQL.js

```diff
@@ -0,0 +1,84 @@
+const request = require('supertest')
+// const app = require('../../app')
+const {
+  buildQueryForData,
+  queryForData,
+  buildParametrableQuery,
+} = require('../../routes/queryGraphQL')
+var assert = require('assert')
+const { gql } = require('graphql-request')
+const expect = require('chai').expect
+
+const mockFrictionlessTableSchema = {
+  fields: [
+    {
+      name: 'float_column',
+      type: { name: 'float8' },
+    },
+    {
+      name: 'int_column',
+      type: { name: 'Int' },
+    },
+    {
+      name: 'text_column',
+      type: { name: 'String' },
+    },
+    {
+      name: 'time_column',
+      type: { name: 'timestamp' },
+    },
+  ],
+}
+
+describe('graphqlQueryBuilder', function () {
+  // describe('function queryForData', function () {
```

Comment:
Are these outdated tests and should be removed?

Reply:
Done